src/preferDecoratorInlineRule.ts

```diff
@@ -0,0 +1,44 @@
+import { IRuleMetadata, Replacement, RuleFailure, Rules } from 'tslint';
+import { isSameLine } from 'tsutils';
+import { Decorator, Node, PropertyAccessExpression, SourceFile } from 'typescript';
+import { NgWalker } from './angular/ngWalker';
+
+export class Rule extends Rules.AbstractRule {
```

Comment:
It makes sense to allow rule configuration (i.e.blacklist decorators for example).

Reply:
Done.